multisig/proposals/vip-016/vip-016-sepolia.ts

```diff
@@ -0,0 +1,176 @@
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setTokensDistributionSpeed(address[],uint256[])", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setMaxTokensDistributionSpeed(address[],uint256[])", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "setMaxLoopsLimit(uint256)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "pauseFundsTransfer()", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME_LIQUIDITY_PROVIDER, "resumeFundsTransfer()", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateAlpha(uint128,uint128)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "updateMultipliers(address,uint256,uint256)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setStakedAt(address[],uint256[])", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "addMarket(address,address,uint256,uint256)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setLimit(uint256,uint256)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "setMaxLoopsLimit(uint256)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "issue(bool,address[])", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "burn(address)", GUARDIAN],
+    },
+    {
+      target: ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [PRIME, "togglePause()", GUARDIAN],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
+    {
+      target: COMPTROLLER,
+      signature: "setPrimeToken(address)",
+      params: [PRIME],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "initializeTokens(address[])",
+      params: [[ETH, BTC, USDC, USDT]],
+    },
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setTokensDistributionSpeed(address[],uint256[])",
+      params: [
+        [ETH, BTC, USDC, USDT],
+        [0, 0, 0, 0],
+      ],
+    },
```

Comment:
I wouldn't add these commands, because we are adding them in the Ethereum TX ([here](https://github.com/VenusProtocol/vips/pull/194/files#diff-56b26dca8ac6aa293e0d9343a53ec641be100a6ab3814ac1ca4c9e0b5d4881b2))